drivers/storage/portworx/deployment.go

```diff
@@ -1319,6 +1321,12 @@ func (t *template) getVolumeMounts() []v1.VolumeMount {
 		t.getBottleRocketVolumeInfoList,
 		t.GetVolumeInfoForTLSCerts,
 	}
+
+	pxVersion := pxutil.GetPortworxVersion(t.cluster)
+	if pxVersion.LessThan(supportedPxVersion) {
+		extensions = append(extensions, t.getCSIVolumeInfoList)
+	}
+
```

Comment:
The CSI volumes were never part of the volume mount list. Why are we trying to add them now?